### DISCUSSION TITLE
Removing film-grain-synthesis during playback

### DISCUSSION THREAD

**Discussion by Daud01** [mpv-player/mpv#14407] [Q&A] [answered]
Hi Is there a way to remove film-grain-synthesis of AV1 during playback in mpv? 
The reason I want to do this is because I want to apply my own custom grain shader+sharpening during playback. And the grain generated from FGS looks ugly after a sharpening filter is applied to it.

**Comment by ghost** [comment, marked as answer]
I haven't tested it but I would assume it's `--vf=format:film-grain=no`

ref: https://mpv.io/manual/master/#video-filters-%3Cfilm-grain

**Reply by Daud01** [reply]
Works perfectly. Thanks!
